koku/reporting/models.py

```diff
@@ -186,6 +194,18 @@
     OCPCostSummaryByNode,
 )
 
+# These are partitioned tables
+OCP_ON_ALL_PERSPECTIVES = (
```

Comment:
nitpick: purely for the sake of consistency with how we've done AWS and OCP tables, let's move this list of models into the `koku/reporting/provider/all/openshift/models.py` file, and import from there.

Reply:
@maskarb 
It should be done now

Reply:
At least it's consistent as this file has tuples of models and the `models.py` file has tuples of table names